lib/fluent/plugin/out_cloudwatch_logs.rb

```diff
@@ -261,7 +261,7 @@ def create_log_stream(group_name, stream_name)
     def log_group_exists?(group_name)
       if @sequence_tokens[group_name]
         true
-      elsif @logs.describe_log_groups.any? {|page| page.log_groups.any? {|i| i.log_group_name == group_name } }
+      elsif @logs.describe_log_groups({log_group_name_prefix: group_name}).log_groups.any? {|i| i.log_group_name == group_name }
```

Comment:
oops, I commented wrong suggestion...this doesn't consider pagination.
How about the following?

```ruby
elsif @logs.describe_log_groups(log_group_name_prefix: group_name).any? {|page| page.log_groups.any? {|i| i.log_group_name == group_name } }
```